tests/fixtures.py

```diff
@@ -144,3 +153,19 @@ def create_rbd_pod(request):
         interface_type=constants.CEPHBLOCKPOOL,
         pvc_name=class_instance.pvc_obj.name
     )
+
+
+@pytest.fixture()
+def delete_pod(request):
```

Comment:
Elad I see that @shylesh wanted to implement those fixtures here:  https://github.com/red-hat-storage/ocs-ci/pull/339/files#diff-10993bfc055d543860f899ca9733c55aR20

And I suggested him to move it under fixtures.py and I think that there is better approach with hasattr cause if setup fail from some reason the teardown fixture will just pass cause there won't be any object to delete WDYT?

Reply:
thanks @petr-balogh, added hasattr

Reply:
Ok, so If this comes in first I will rebase on this.